### ISSUE TITLE
Cannot assign to read only property 'rotate' of object

### ISSUE THREAD

**Issue by jonnycyborg** [TheLastProject/lovelace-valetudo-map-card#35]
Lovelace card states "Cannot assign to read only property 'rotate' of object '#<Object>'" when setting the rotate value to any number other than 0.

Example config:
```
type: 'custom:valetudo-map-card'
entity: sensor.robovac_map
rotate: 180
crop:
  top: 0
  bottom: 0
  left: 0
  right: 0
min_height: 0
```
Interestingly, `rotate: 3.14rad` works fine.

I believe this started showing up in Home Assistant v0.106.1 which might help either narrow the issue or mean it's not an issue on your end. I'm also not 100% sure if/how the setConfig() function has changed throwing the new error, but would love some help tracking it down.

**Comment by TheLastProject** [comment]
@zvldz reported this in #34 but I thought they were using an old version of the card due to the recent fixing of #32.

However, after I upgraded from 0.106.0 to 0.106.1 today, I can now reproduce this issue. So Home Assistant change the behaviour not just in 0.106.0, but again in 0.106.1.

Looking at it a bit more, I'm pretty sure it's cause by this line:
https://github.com/TheLastProject/lovelace-valetudo-map-card/blob/becff16cc9234b49ef0f42c881671f0f52e6c28b/valetudo-map-card.js#L239

Because `Number(0)` will return `0`, this causes it to be seen as not a number in the if check (whoops!), making 0 not break anything.

I'll use this ticket to track the issue instead of #34 due to this being the more detailed report.

**Comment by TheLastProject** [comment]
Fixed in 7db9b89e85d1ddd5f6a573509cc5ee793a867fb1.

**Comment by jonnycyborg** [comment]
Awesome that worked great! Thanks for the quick fix!
